src/main/java/racingcar/RacingCarService.java

```diff
@@ -0,0 +1,58 @@
+package racingcar;
+
+import java.util.List;
+import org.springframework.stereotype.Service;
+import racingcar.dto.RacingCarNamesRequest;
+import racingcar.dto.RacingCarStatusResponse;
+import racingcar.dto.RacingCarWinnerResponse;
+import racingcar.repository.GameRepository;
+import racingcar.repository.PlayerRepository;
+import racingcar.service.RacingCarGame;
+import racingcar.service.RandomMoveStrategy;
+import racingcar.service.TryCount;
+
+@Service
+public class RacingCarService {
+
+    private final GameRepository gameRepository;
+    private final PlayerRepository playerRepository;
+
+    public RacingCarService(final GameRepository gameRepository, final PlayerRepository playerRepository) {
+        this.gameRepository = gameRepository;
+        this.playerRepository = playerRepository;
+    }
+
+    public PlayResponse play(final PlayRequest playRequest) {
+        RacingCarGame racingCarGame = new RacingCarGame();
+        createCars(playRequest, racingCarGame);
+        TryCount tryCount = new TryCount(playRequest.getCount());
+        playGame(tryCount, racingCarGame);
+        PlayResponse response = save(playRequest.getCount(), racingCarGame);
+        return response;
```

Comment:
```suggestion
        return save(playRequest.getCount(), racingCarGame);
```

지금도 좋지만, 인라인으로 처리할 수도 있겠네요.

Reply:
사실 `save` 메서드가 조회와 명령을 동시에 실행하고 있는 메서드라고 생각하고 있습니다.
저는 인라인으로 처리하게 되면 `save`가 리턴하고 있는 값이 명시적으로 보이지 않는다고 생각해서 의도적으로 인라인으로 처리하지 않았습니다 🤔

다니의 생각은 어떠신가요? 다른 개발자가 리턴 타입으로도 충분히 유추 가능할까요?